uniswap-v2-core/contracts/UniswapV2Pair.sol

```diff
@@ -35,6 +47,27 @@ contract UniswapV2Pair is IUniswapV2Pair, UniswapV2ERC20 {
         unlocked = 1;
     }
 
+    modifier onlyFactory() {
+        require(msg.sender == factory, 'Vexchange: FORBIDDEN');
+        _;
+    }
+
+    function setRecoverer(address _recoverer) external onlyFactory {
+        recoverer = _recoverer;
+    }
+
+    function setPlatformFee(uint _platformFee) external onlyFactory {
+        require(_platformFee < MAX_PLATFORM_FEE, "Vexchange: INVALID_PLATFORM_FEE");
+
+        platformFee = _platformFee;
+    }
+
+    function setSwapFee(uint _swapFee) external onlyFactory {
+        require(_swapFee > MIN_SWAP_FEE && _swapFee < MAX_SWAP_FEE, "Vexchange: INVALID_SWAP_FEE");
```

Comment:
Align this `require` check with the factory. Either make the factory all in one line, or split this `require` over two lines